README.md

```diff
@@ -1,3 +1,37 @@
 # Monitoring operator for storages
 
 This operator takes care of deployment of mixins of storage system in Openshift and Kubernetes. The mixins are the mechanism for adding alerting rules, grafana dashboards and recording rules for prometheus in Openshift and Kubernetes.
+
+# Deploy Operator with mixins
+To deploy the operator simply deploy to your cluster. The set of steps to be followed
+for deployment of monstorak operator and related artifacts, follow the below steps
+
+## Create the namespace `storage-monitoring`
+```
+oc create -f deploy/kubernetes/01_namespace.yaml
```

Comment:
I don't see the kubernetes folder inside deploy right now.

Reply:
Dependent on https://github.com/monstorak/monstorak/pull/24